lib/components/Checkbox/index.tsx

```diff
@@ -20,17 +24,16 @@ export const Checkbox = forwardRef(
       label,
       labelClassName,
       wrapperClassName,
+      name,
       ...rest
     }: CheckboxProps,
     ref: Ref<HTMLInputElement>
   ) => {
-    const id = useId();
-
     return (
       <div className={clsx("deriv-checkbox", wrapperClassName)}>
         <div className="deriv-checkbox__wrapper">
           <input
-            id={rest.id ?? id}
+            id={rest.id ?? name}
```

Comment:
```suggestion
            id={rest.id ?? (name + Date.now().toString())}
```
Since useId requires React 18, and you need a unique identifier, you can append the component name to Date.now() to get some amount of uniqueness from the id

Reply:
Willl implement this in the next fix then 😂